### PR TITLE
Fix issue #7: add default values for relationshipTypes custom field g…

### PR DESCRIPTION
**Pull request by lucky091588**
…enerate tokens

### CODE REVIEW

File: reltoken.php [hunk 1]
```diff
@@ -247,6 +247,24 @@ function reltoken_civicrm_install() {
...
+    'return' => 'id',
+  ]);
+
+  // add generating tokens in RelationshipType
+  $result = civicrm_api3('RelationshipType', 'get', [
+    'sequential' => 1,
+    'is_active' => 1,
+    'api.RelationshipType.create' => [
+      'name_a_b' => '$value.name_a_b',
+      'name_b_a' => '$value.name_b_a',
```
Comment (twomice) [inline reltoken.php]: @lucky091588 Seems redundant to set `name_a_b` and `name_b_a`. Is that necessary?

Comment (lucky091588) [reply reltoken.php]: @twomice It is necessary since it is a mandatory field when using RelationshipType create in api3.

Comment (lucky091588) [reply reltoken.php]: @twomice I just confirmed that it's not necessary since it has the `id` with a value which means updating.

**Comment by twomice** [comment]
Merged, thanks @lucky091588

**Pull request merged**